core/src/testimony_recorder.cpp

```diff
@@ -38,9 +41,9 @@ void AOClient::addStatement(QStringList packet)
             }
         }
         else if (area->testimonyRecording() == AreaData::TestimonyRecording::ADD) {
-               packet[14] = "1";
-               area->addStatement(c_statement, packet);
-               area->setTestimonyRecording(AreaData::TestimonyRecording::PLAYBACK);
+                packet[14] = "1";
```

Comment:
I'll have to slap a "reject all commits where the author couldn't be arsed to clang-format" CI action up here sometime later.

Reply:
this tbh